### PR TITLE
Refactor traceback errors

### PR DESCRIPTION
**Pull request by antgurevich**
- Uncommented `events` cog (events must be added to .json before deploying to prod or running pr on local)
- Changed traceback errors

### CODE REVIEW

File: Cogs/events.py [hunk 1]
```diff
@@ -49,7 +48,7 @@ async def clear_error(self, ctx, error):
             print(f"non-admin {ctx.message.author} tried to use createevent")
         else:
             await ctx.send(embed = discord.Embed(title = "Unknown error. Please contact developers to check logs", color = discord.Color.red()))
-            print("Create Event error: ", error, error.withtraceback())
+            raise Exception("Create Event error: ", error)
```
Comment (DMLooter) [inline Cogs/events.py:51]:
```suggestion
            print("Create Event error: ", error)
            raise error
```

Comment (antgurevich) [reply Cogs/events.py:51]: Ohh ok, thought your comment in the issue post was just generic pseudocode not the actual code

**Review by antgurevich — COMMENTED** [review]
File might get deleted anyways if we don't need it anymore (per Issue question I just left)

**Review by DMLooter — APPROVED** [review]
Perfect!

**Pull request merged**